### PR TITLE
Add tests and release automation

### PR DESCRIPTION
**Pull request by robmosca**
Adding Jests tests, fixing a minor bug and also adding release automation through a GitHub workflow.

### CODE REVIEW

File: .changeset/plenty-nails-lie.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"import-map-overrides": patch
+---
+
+Add automation to publish package
```
Comment (namelivia) [inline .changeset/plenty-nails-lie.md:5]: What is this file for? Im curious

Comment (robmosca) [reply .changeset/plenty-nails-lie.md:5]: This is the changeset for this commit. You accumulate changesets file until you run `changeset version` which updates the version and the changelog and removes the changeset files.

File: package.json [hunk 2]
```diff
@@ -3,21 +3,24 @@
...
+    "check-format": "prettier --check src",
     "clean": "rimraf dist",
     "watch": "cross-env NODE_ENV=development rollup -c --watch",
     "copy-test-files": "copyfiles test/**/* dist -f",
-    "test-serve": "serve .",
-    "test-embed-serve": "serve . -l 3333",
-    "test": "concurrently pnpm:watch pnpm:copy-test-files pnpm:test-serve pnpm:test-embed-serve",
+    "serve": "serve .",
+    "embed-serve": "serve . -l 3333",
+    "test": "jest",
```
Comment (namelivia) [inline package.json:18]: Much better!

File: src/api/js-api.js [hunk 3]
```diff
@@ -207,7 +207,13 @@ function init() {
...
                 nextPromise = fetchExternalMap(scriptEl.src);
               } else {
-                nextPromise = Promise.resolve(JSON.parse(scriptEl.textContent));
+                let parsedContent;
+                try {
+                  parsedContent = JSON.parse(scriptEl.textContent);
+                } catch (e) {
+                  parsedContent = createEmptyImportMap();
+                }
+                nextPromise = Promise.resolve(parsedContent);
```
Comment (namelivia) [inline src/api/js-api.js:216]: Was this a bug in the original code?

**Review by namelivia — APPROVED** [review]
Lgtm! :rocket:

File: src/api/js-api.js (hunk 3 above)
Comment (robmosca) [reply src/api/js-api.js:216]: It was indeed:
https://github.com/single-spa/import-map-overrides/pull/87/files#r1264723158

Comment (robmosca) [reply src/api/js-api.js:216]: More of an unwanted behaviour.

**Pull request merged**